.github/workflows/l10n.yml

```diff
@@ -18,10 +18,10 @@ jobs:
     steps:
       - run: echo "${{ github.actor }}"
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      - name: Use Node.js 18.x
-        uses: actions/setup-node@1a4442cacd436585916779262731d5b162bc6ec7 # v3.8.2
+      - name: Use Node.js 20.x
+        uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4.0.1
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
Bumping node to v20, because all the actions have switched to using v20 on their latest majors.